readme.md

```diff
@@ -504,6 +504,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Free Software](https://github.com/johnjago/awesome-free-software) - Free as in freedom.
 - [Framer](https://github.com/podo/awesome-framer) - Prototyping interactive UI designs.
 - [Markdown](https://github.com/BubuAnabelas/awesome-markdown) - Markup language.
+- [Dev Fun](https://github.com/mislavcimpersak/awesome-dev-fun) - Libs/packages/languages that have no real purpose but to make a developer chuckle.
```

Comment:
updated in https://github.com/sindresorhus/awesome/pull/914/commits/8aa8a748c26e33aa07bc148e5b1f77e551272aa5